packages/fx-core/templates/plugins/resource/apim/bicep/output.template.bicep

```diff
@@ -0,0 +1,6 @@
+output apimServiceResourceId string = apimProvision.outputs.serviceResourceId
+output apimProductResourceId string = apimProvision.outputs.productResourceId
+output apimAuthServiceResourceId string = apimConfiguration.outputs.authServiceResourceId
+output apimPublisherEmail string = apimPublisherEmail
```

Comment:
can we do not output this information? apim plugin can set related context in its own code directly.
This may also related to a feature decision: if user modifies publisher email/name, should we persist it to profile.dev.json? My personal preference is no, since user do not need to review it, and we do not need these value either. Can you help talk with Junjie on this?

Reply:
Will sync with @MuyangAmigo and @XiaofuHuang and fix in other PR if any changes. Thx